CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 CHANGELOG
 =========
+3.11.0
+------
+
+**BUG FIXES**
+- Remove requirement of outbound ports on VPC security groups for FSx for OpenZFS
```

Comment:
1. If possible, let's specify which ports are not required anymore.
2. "for FSx for OpenZFS" => "for OpenZFS"